islandora_videojs.module

```diff
@@ -64,9 +66,12 @@ function template_preprocess_islandora_videojs(array &$variables) {
   if (isset($params['sources']) && is_array($params['sources'])) {
     $variables['sources'] = $params['sources'];
   }
-  else {
+  elseif (isset($params['url']) && isset($params['mime'])) {
     $variables['sources'][] = array('url' => $params['url'], 'mime' => $params['mime']);
   }
+  else {
+    $variables['sources'] = array();
```

Comment:
Fair enough, I'm not tied to that text. I don't mind showing the player, I just want people to be aware that there is nothing to play on this object.